### DISCUSSION TITLE
have a rows read/rows returned on insights table

### DISCUSSION THREAD

**Discussion by tonyaellie** [planetscale/discussion#481] [Feature Requests]
This would make it easier to find which queries need to be optimised.

**Comment by orware** [comment]
Hi @tonyaellie,

I may be misunderstanding your suggestion, but we do already have columns that show the rows read and rows returned information within the Insights table output today:
https://planetscale.com/docs/concepts/query-insights#queries-list

If you'd like to share some more detail though that may be helpful if there is something in the current output that may be missing?
